### ISSUE TITLE
Been waiting forling time for stats

### ISSUE THREAD

**Issue by FavioVazquez** [hstove/issue_stats#37]
Hi, I've use issuestats before for one repo and everything went fine, but I've been waiting for more than one day to have the stats for another repo. It's a very simple one and doesn't have lots of data or issues, so I found it weird. 

![screenshot from 2015-11-29 19 20 59](https://cloud.githubusercontent.com/assets/10162068/11461708/57d92792-96ce-11e5-9e56-c7af41f29f74.png)

Thanks.

**Comment by ChristopherKing42** [comment]
:+1: At least an estimate of how long it would take would be good.

**Comment by hstove** [comment]
Hm there is something wrong here, looking into it! Sorry for the really long delay.

**Comment by FavioVazquez** [comment]
Yes, it's weird. If I open the page, http://issuestats.com/github/FavioVazquez/DeteccionRayosCosmicos-PCF, it's still telling me to hang tight haha. If you have a solution please let me know.

**Comment by SteveVanOpstal** [comment]
It seems to me the angular issuestats are also not updating: http://issuestats.com/github/angular/angular
Now I've refreshed the queue request again but before it was displaying: 'Enqueued to be updated 6 days ago'

**Comment by matkoniecz** [comment]
rails/rails was at 28 days...

Together with application errors (#46) it seems to be infrastructure problem - and to fix it somebody would need to pay more for servers.
